src/ImageSharp/PixelFormats/PixelOperations{TPixel}.cs

```diff
@@ -159,5 +161,45 @@ public partial class PixelOperations<TPixel>
 
             PixelOperations<TDestinationPixel>.Instance.From(configuration, sourcePixels, destinationPixels);
         }
+
+        /// <summary>
+        /// Bulk operation that packs 3 seperate RGB channels to <paramref name="destination"/>.
+        /// The destination must have a padding of 3.
```

Comment:
We expect `destination` to be padded, because of quirks of the Rgb24 AVX2 implementation. This will need some extra trickery (a copy) when using it arond the last line(s) of Jpeg, but that shouldn't be too hard.

Reply:
Yeah when I did the slice from 4 to 3channels I offset the write so there wasn't any padding required but that could only be done with SSE so likely wouldn't be fast enough here.

Reply:
The problem is that I'm doing overlapped stores of 256 byte vectors in the following format:

```
rgb1   = <R00, G00, B00, R01, G01, B01, R02, G02 | B02, R03, G03, B03, R04, G04, B04, R05 || B05, G05, R06, G06, B06, R07, G07, B07 | ___, ___, ___, ___, ___, ___, ___, ___>
rgb2   = <R08, G08, B08, R09, G09, B09, R10, G10 | B10, R11, G11, B11, R12, G12, B12, R13 || B13, G13, R14, G14, B14, R15, G15, B15 | ___, ___, ___, ___, ___, ___, ___, ___>
rgb3   = <R16, G16, B17, R17, G17, B17, R18, G18 | B18, R19, G19, B19, R20, G20, B20, R21 || G21, B21, R22, G22, B22, R23, G23, B23 | ___, ___, ___, ___, ___, ___, ___, ___> 
rgb4   = <R24, G24, B24, R25, G25, B25, R26, G26 | B26, R27, G27, B27, R28, G28, B28, R29 || G29, B29, R30, G30, B30, R31, G31, B31 | ___, ___, ___, ___, ___, ___, ___, ___>  
```
Storing `rgb4` would result in an overflow, since `B31` is the last element.

Reply:
Would it perhaps be better to vectorize one less round than max and slice accordingly after? I'm concerned about the padding requirement.

Reply:
That's something we can do a good solution on the call-site yeah ... but the input span has to be still padded, there is no other way for us to make sure there is no overflow within the `PackFromRgbPlanes` method.

Idea:
- Define a utility `Buffer2D.TryGetPaddedRowSpan(y, padding)`, which will return `true` for everything except the last row, or last few rows for the `width < padding` corner case.
- If there is no padding do an automatic copy in `PackFromRgbPlanes()` instead of throwing an exception. (Slow(er) path for the last row)

Reply:
Sounds good to me!